searx/engines/google.py

```diff
@@ -118,10 +118,10 @@
 
 # in the result group there is <div class="r" ../> it's first child is a <a
 # href=...> (on some results, the <a> is the first "descendant", not ""child")
-href_xpath = './/div[@class="r"]//a/@href'
+href_xpath = './/div[@class="rc"]//a/@href'
 
 # in the result group there is <div class="s" ../> containing he *content*
-content_xpath = './/div[@class="s"]'
+content_xpath = './/div[@class="rc"]/div[2]/div/span'
```

Comment:
Only the text, the additional data are are ignored (key/value, internal links, etc...).